service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/respondentsolicitor/RegisteredRespondentSolicitorContentProvider.java

```diff
@@ -5,24 +5,36 @@
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
 import uk.gov.hmcts.reform.fpl.model.RespondentSolicitor;
 import uk.gov.hmcts.reform.fpl.model.notify.respondentsolicitor.RegisteredRespondentSolicitorTemplate;
+import uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper;
 
+import static java.util.Objects.isNull;
 import static org.apache.commons.lang3.StringUtils.EMPTY;
-import static org.apache.commons.lang3.StringUtils.isBlank;
+import static org.apache.logging.log4j.util.Strings.isBlank;
 
 @Component
 @RequiredArgsConstructor(onConstructor_ = {@Autowired})
 public class RegisteredRespondentSolicitorContentProvider {
 
+    private static final String MANAGE_ORG_URL = "https://manage-org.platform.hmcts.net";
```

Comment:
If you hardcode in a production URL will this still work with AAT environment?

Reply:
On second thoughts, as this is just a notification it isn't going to have much impact on testing in AAT, will be easy to work around.